cmd/node/config/config.toml

```diff
@@ -34,7 +34,7 @@
    # SetGuardianEpochsDelay represents the delay in epochs between the execution time of the SetGuardian transaction and
    # the activation of the configured guardian.
    # Make sure that this is greater than the unbonding period!
-   SetGuardianEpochsDelay = 20
+   SetGuardianEpochsDelay = 2 # for mainnet should be 20, 2 is just for testing
```

Comment:
should this be modified ?

Reply:
yes here it can be 1 (flags compression will come with the merge from rc/2022-july and then it will match the other activations)
but when we add the flag to mainnet config, we need to take care and change it to 20.